RELEASE.md

```diff
@@ -1,5 +1,3 @@
-
 # Release Process
-1. Bump version in pynitrokey/VERSION. Use 0.x.y format.
-2. Make an annotated tag in format v0.x.y.nitrokey, with message describing major change.
-3. Run CI release to release it
+1. Bump version in pynitrokey/VERSION. Use `0.x.y` format.
+2. Create a release in the Github UI with the tag in format `v0.x.y`. The CD pipeline will run automatically on a new release.
```

Comment:
Can we keep step 2 so that we have signed tags for the releases?

Reply:
Yes, this is more or less a matter of which process you prefer.
You have two options to make a release:

1. You create a signed tag locally and push it to the repository. Afterwards picking the tag in the release UI on Github.
2. You create the release and tag through the release UI on Github. If you configured Github to sign your commits/tags it will sign this tag automatically. The description entered in the release UI will become the annotation of the release tag.

The important action is to create a release, because only this event will trigger the pipelines.